java/gradle/dse/build.gradle

```diff
@@ -0,0 +1,33 @@
+group 'com.datastax.spark.example'
+version '0.1'
+apply plugin: 'java'
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://datastax.artifactoryonline.com/datastax/public-repos/"
+    }
+}
+
+configurations {
```

Comment:
May want to add a quick comment "The provided configuration behaves the same as the sbt "provided" keyword which will cause jars to be excluded from assembled fat-jar"